### ISSUE TITLE
Type returned by Moralis.Units.Token should be BigNumber, not string

### ISSUE THREAD

**Issue by Vivraan** [MoralisWeb3/Moralis-JS-SDK#191]
### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/SECURITY.md).
- [x] I have searched through [existing issues](https://github.com/MoralisWeb3/Moralis-JS-SDK/issues?q=is%3Aissue) and the [Moralis Forum](https://forum.moralis.io/).
- [x] I can reproduce the issue with the [latest SDK release](https://github.com/MoralisWeb3/Moralis-JS-SDK/releases), and my server is updated to the latest version.

### Issue Description

When I use the typescript version of the Moralis SDK, I find that the type definition of `Moralis.Units.Token` is
```ts
  class Units {
    static ETH: (value: number | string) => string;
    static Token: (value: number | string, decimals: number) => string;
    static FromWei: (value: number | string, decimals: number) => number;
  }
```

Whereas it should be
```ts
class Units {
    // ...
    static Token: (value: number | string, decimals: number) => BigNumber;
    // ...
}
```

### Steps + code to reproduce

1. `npm install moralis`
2. Create a typescript file with some bare code using `Moralis.Units.Token()`.

### Actual Outcome

The IDE thinks it's a string.

### Expected Outcome

The IDE should tell me it's a `BigNumber`.
### Environment

Server

- Moralis server version: `0.0.326` (irrelevant)

Client

- Moralis SDK version: `0.0.183`
- Operating system: `Windows 11 Home 21H2 (build 22000.434)`
- Browser: `Brave 1.34.80 Chromium: 97.0.4692.71 (Official Build) (64-bit)`

### Logs

None.

**Comment by ErnoW** [comment]
This has been resolved in v1.0.0

**Comment by Vivraan** [comment]
> This has been resolved in v1.0.0

![image](https://user-images.githubusercontent.com/12777564/151154751-29c3f47d-5d9b-4d3f-8683-9f486598d7bd.png)

This is for `moralis@1.1.0`.

**Comment by ErnoW** [comment]
> > This has been resolved in v1.0.0
> 
> ![image](https://user-images.githubusercontent.com/12777564/151154751-29c3f47d-5d9b-4d3f-8683-9f486598d7bd.png)
> 
> This is for `moralis@1.1.0`.

That's correct, right? Typescript is now corresponding with the implementation. Both return a string.

**Comment by Vivraan** [comment]
> > > This has been resolved in v1.0.0
> > 
> > 
> > ![image](https://user-images.githubusercontent.com/12777564/151154751-29c3f47d-5d9b-4d3f-8683-9f486598d7bd.png)
> > This is for `moralis@1.1.0`.
> 
> That's correct, right? Typescript is now corresponding with the implementation. Both return a string.

No, as per the docs and the developer discussion on Discord it should return a `BigNumber`.

**Comment by Vivraan** [comment]
Okay, I just checked the docs and it seems to return a string appended with ` Wei` in the end. I was wrong about this one.
